parser-typechecker/src/Unison/Codebase/Editor/Output.hs

```diff
@@ -175,6 +175,7 @@ data Output v
                (Map Reference (DisplayThing (Decl v Ann)))
                (Map Reference (DisplayThing (Term v Ann)))
   | MetadataMissingType PPE.PrettyPrintEnv Referent
+  | MetadataNotFound (Maybe Text)
```

Comment:
Maybe better to use the provided `HQ` in the message, since `HQ.toName` throws away part of their query — wdyt?

Reply:
I had not realised that I could `P. HQ` without ending up with a `Maybe Name`, taking care of it. :+1: